### PR TITLE
chore: update karma-and-qunit

### PR DESCRIPTION
**Pull request by brandonocasey**
## Description
Update karma and qunit to see if browserstack timeouts stop

### CODE REVIEW

**Comment by gkatsev** [comment]
Also, this was a thing https://github.com/videojs/http-streaming/pull/283/files

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
 before_install:
   - npm install -g greenkeeper-lockfile@1
 before_script:
-  # pulseaudio is needed for dockerized chrome since it has a dummy sink and
-  # the dummy audio ALSA kernel module isn't loaded by Travis
-  # libavcodec54 is needed by firefox for h.264 support
-  - sudo apt-get install --no-install-recommends pulseaudio libavcodec54
-  - pulseaudio -D
+  # libav-tools is needed by firefox for h.264 support
+  - test -z BROWSER_STACK_ACCESS_KEY && sudo apt-get update && sudo apt-get install libav-tools || echo "skipping"
```
Comment (brandonocasey) [inline .travis.yml]: libavcodec54 is now in libav-tools only, pulseaudio isn't needed anymore.

File: test/test-helpers.js [hunk 2]
```diff
@@ -287,7 +287,15 @@ export const mockTech = function(tech) {
 
   tech.play_ = tech.play;
   tech.play = function() {
-    tech.play_();
+
+    const playPromise = tech.play_();
+
```
Comment (brandonocasey) [inline test/test-helpers.js]: rejected promises have to be handled in newer qunits

File: test/segment-transmuxer.test.js [hunk 3]
```diff
@@ -53,6 +54,7 @@ QUnit.module('Segment Transmuxer', {
     assert.timeout(5000);
   },
   afterEach(assert) {
+    dispose();
```
Comment (brandonocasey) [inline test/segment-transmuxer.test.js]: Not sure how this didn't come up earlier, maybe qunit is  faster now. Basically we have to cleanup the segment-transmuxer queue after each test.

**Pull request merged**